### PR TITLE
:bug: Topic regular error for the non wildcard configuration

### PR DESCRIPTION
**Pull request by yanmxa**
## Related issue(s)

Fixes #https://issues.redhat.com/browse/ACM-13161

### CODE REVIEW

File: test/integration/operator/hubofhubs/transporter_test.go [hunk 1]
```diff
@@ -187,6 +196,8 @@ var _ = Describe("transporter", Ordered, func() {
 	})
 
 	It("should pass the strimzi transport configuration", func() {
+		Skip("skip the ....2")
```
Comment (clyang82) [inline test/integration/operator/hubofhubs/transporter_test.go:199]: debug code?

File: operator/pkg/controllers/hubofhubs/transporter/protocol/strimzi_kafka_controller.go [hunk 2]
```diff
@@ -17,30 +25,55 @@ import (
...
+	if err := r.GetClient().Get(ctx, config.GetMGHNamespacedName(), mgh); err != nil {
 		if errors.IsNotFound(err) {
 			return ctrl.Result{}, nil
 		}
 		return ctrl.Result{}, err
 	}
 
-	if err := r.reconcileResources(ctx, mgh); err != nil {
+	// kafka metrics, monitor, global hub kafkaTopic and kafkaUser
+	if err := r.renderKafkaResources(mgh); err != nil {
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/transporter/protocol/strimzi_kafka_controller.go:53]: I think you may create kafkaResources after `NewStrimziTransporter`

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: test/integration/operator/hubofhubs/transporter_test.go (hunk 1 above)
Comment (yanmxa) [reply test/integration/operator/hubofhubs/transporter_test.go]: Rmoved

**Comment by yanmxa** [comment]
e2e setup error, fix this in another PRs

/test test-e2e

**Comment by yanmxa** [comment]
Verified in the OCP. retrying:
/test test-e2e

**Pull request merged**